### DISCUSSION TITLE
Review of Pixabay

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#325] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/pixabay.svg">

**Comment by GnuLinuxIsCool** [comment]
Super useful! we could not make a plugin that takes care of all the banks images and icons in a line, with the ability to filter by bank?

**Reply by Misokados** [reply]
Hello,
Please describe your idea as detailed as possible, we will check it out.
